src/assets/css/ProfilePage.css

```diff
@@ -0,0 +1,64 @@
+.container {
+    display: flex;
+}
+
+.some-page-wrapper {
+    display: flex;  
+    flex-direction: column;
+    width: 100%;
+    
+}
+
+.column{
```

Comment:
Nit: This class is technically a container, consider changing the name to something that reflects this